test/services/rules.service.test.js

```diff
@@ -0,0 +1,22 @@
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+const { describe, it } = exports.lab = Lab.script()
+const { expect } = Code
+const nock = require('nock')
+
+const RulesService = require('../../app/services/rules.service')
+const RulesServiceHelper = require('../support/helpers/rules.service.helper')
+
+describe('Rules service', () => {
+  it('calls the service', async () => {
```

Comment:
I'm a strong advocate that tests are there to do a few things

- check it works
- help us write cleaner code
- document how it's expected to behave

If you agree with that last point it sometimes means you'll write additional tests to help explain the behaviour even though coverage doesn't change.

I think the case applies in this scenario. I'd like to see a couple of tests that highlight

- the endpoint we call is based on regime
- the endpoint we call is also based on financial year